src/utils/SpokeUtils.ts

```diff
@@ -57,6 +57,33 @@ export function populateV3Relay(
   return spokePool.populateTransaction.fillV3Relay(v3RelayData, repaymentChainId, { from: relayer });
 }
 
+/**
+ * Concatenate all fields from a Deposit, Fill or SlowFillRequest into a single string.
+ * This can be used to identify a bridge event in a mapping. This is used instead of the actual keccak256 hash
+ * (getRelayDataHash()) for two reasons: performance and the fact that only Deposit includes the `message` field, which
+ * is required to compute a complete RelayData hash.
+ * note: This function should _not_ be used to query the SpokePool.fillStatuses mapping.
+ */
+export function getRelayEventKey(data: RelayData & { destinationChainId: number }): string {
```

Comment:
I like this name so it's not as easily confused with the keccak256 hash used by the contracts